src/main/webapp/components/Navigation.js

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+
+/**
+ * Main List Element to be displayed on the Navigation.
+ *
+ * @param {href: a URL String to a resource, innerText: The display text}
+ */
+const NavigationItem = props =>
```

Comment:
```suggestion
const NavigationItem = (props) =>
```
(to keep it consistent with `(props)` below

Reply:
Updated here c3c481f